src/styles/styles.js

```diff
@@ -2214,6 +2214,15 @@ const styles = {
         flex: '0 0 auto',
     },
 
+    absolute: {
+        position: 'absolute',
+    },
+
+    iconInlineWrapper: {
+        overflow: 'visible',
+        backgroundColor: 'transparent',
+    },
+
```

Comment:
Not needed.
https://github.com/Expensify/App/blob/2aaa8fd10bec0cfb6346140595a5f55540edacd9/src/styles/styles.js#L287
https://github.com/Expensify/App/blob/2aaa8fd10bec0cfb6346140595a5f55540edacd9/src/styles/utilities/overflow.js#L13
https://github.com/Expensify/App/blob/2aaa8fd10bec0cfb6346140595a5f55540edacd9/src/styles/utilities/positioning.js#L9